doc/src/documentation_style_guide.rst

```diff
@@ -346,87 +349,114 @@ LaTeX Recommendations
+letter α would be written as ``alpha`` in code and $\alpha$ in LaTeX. The
+reason being that $\alpha$ cannot be used in contexts referring to Python code
+because it is not valid Python, and conversely ``alpha`` would be incorrect in
+a math context because it does not render as the Greek letter (α).
+
+**Example**
+
+::
+
+    class loggamma(Function):
+        r"""
+        The ``loggamma`` function implements the logarithm of the gamma
+        function (i.e, $\log\Gamma(x)$).
+
+        """
+
+Variables listed in the parameters after the function name should be italicized
+using Sphinx emphasis with asterisks like ``*this*``.
+
 **Example**
 
 ::
 
-   def stirling(n, k, d=None, kind=2, signed=False):
-       """
-       ...
+    def stirling(n, k, d=None, kind=2, signed=False):
+        """
+        ...
 
-       The first kind of Stirling number counts the number of permutations of
-       ``n`` distinct items that have ``k`` cycles... This counts the ways to
-       partition $n$ consecutive integers into $k$ groups with no pairwise
-       difference less than $d$.
+        The first kind of Stirling number counts the number of permutations of
+        *n* distinct items that have *k* cycles; the second kind counts the
+        ways in which *n* distinct items can be partitioned into *k* parts.
+        If *d* is given, the "reduced Stirling number of the second kind" is
+        returned: $S^{d}(n, k) = S(n - d + 1, k - d + 1)$ with $n \ge k \ge d$.
+        This counts the ways to partition $n$ consecutive integers into $k$
+        groups with no pairwise difference less than $d$.
 
-       """
+        """
 
 Note that in the above example, the first instances of *n* and *k* are
-referring to the input parameters of the function ``stirling``, so they refer
-to Python variables and are therefore formatted as code. The second instances
-of *n* and *k* are talking about mathematical expressions, so they are
-formatted as math.
+referring to the input parameters of the function ``stirling``. Because they
+are Python variables but also parameters, they are formatted as parameters in
+italics. The last instances of $n$ and $k$ are talking about mathematical
+expressions, so they are formatted as math.
 
-In general, the rule of thumb is to consider if the variable in question were
-something that rendered differently in code and in math. For example, the Greek
-letter α would be written as ``alpha`` in code and $\alpha$ in LaTeX. The
-reason being that $\alpha$ cannot be used in contexts referring to Python code
-because it is not valid Python, and conversely ``alpha`` would be incorrect in
-a math context because it does not render as the Greek letter (α).
+If a variable is code, but is also a parameter, the parameter formatting takes
+precedence and it should be rendered in italics. If a variable is code and not
+a parameter, it should be in double backticks.
```

Comment:
This should be clearer that a parameter is only italics if it is by itself. If a parameter appears in a larger code expression it should be rendered as code (backticks).

Reply:
Ok I've added clarification here.